src/components/AllMovies/AllMovies.js

```diff
@@ -1,24 +1,22 @@
-import React, { Component } from 'react'
+import React from 'react'
 import App from '../App/App'
+import Movie from '../Movie/Movie'
 
-class AllMovies extends Component {
```

Comment:
Happy we decided to change AllMovies from a class component to a functional; For the time, seems like it is just going to be displaying all the info of each movie